CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Version History
     * All Version bumps are required to update this file as well!!
 ----
 
+* 18.0.0 Adding job search version 3 as additional call
```

Comment:
Does adding a new call require a major version bump (salute). It's still backwards compatible, right?